docknetwork/dock-app
# Fix credential filter reset button

## Proposed Changes
- A fix to the credentials filter "RESET" button not working when clicked. This was causing users to need to manually reset individual filter parameters.

## Testing instructions
- Set filter values for the filtering fields
- After applying the filter, the credentials list should reflect the filters
- Click on "RESET" button
- The fields should now be reset to default values and the credentials list restored
